### ISSUE TITLE
Update cryptol output in the "Programming Cryptol" book

### ISSUE THREAD

**Issue by brianhuffman** [GaloisInc/cryptol#523]
The book contains a lot of verbatim output, error messages, etc. which were copied from the cryptol REPL. Much of this text is now out of date. We should make a pass over the book to update all of the verbatim REPL output before the next official Cryptol release.

**Comment by atomb** [comment]
This probably won't be a near-term possibility, but in the past I've sometimes set up documentation using markup and tools (usually custom to that particular document, though something general would be nice) that automatically run the program being documented and capture the current output. Would it be worth trying something like that here? In particular, Appendix A has a *lot* of output which will be somewhat tedious to update.

**Comment by atomb** [comment]
I made a pass over it, and I think everything's up to date, with several caveats:
- I did it by hand, so I could have missed things.
- The output sometimes depends on things like `set base=10` without explicitly mentioning it (though after the `:set base` command has been introduced).
- The output in the book sometimes leaves out messages about monomorphization performed by the REPL when printing polymorphic values.
- I didn't go through all the code examples to make sure they still type check. I just focused on the REPL examples.
